processor/processor.go

```diff
@@ -980,6 +980,10 @@ func (proc *HandleT) getFailedEventJobs(response transformer.ResponseT, commonMe
 			failedEvent.StatusCode, commonMetaData.SourceID, commonMetaData.DestinationID, failedEvent.Error,
 		)
 
+		if failedEvent.StatusCode == transformer.DROP_STATUS_CODE {
+			continue
```

Comment:
Are these events reported by the reporting service after this change? If not, should they?

Reply:
They are.
They shouldn't.👍 
Should be part of the diff metric.